src/components/Jobs/Card/Card.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types'
```

Comment:
Please remove the `Jobs` directory and move all components to the components directory and Forms to the Forms directory.

Reply:
done